### PR TITLE
[AC-1286] AC Team codeownership moves - Organization Auth

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Move the remaining Organization Auth files into AC Team code ownership.

Most of these were already done in previous bulk moves, this just tidies up some of the request/response models.

## Code changes

* **file.ext:** Description of what was changed and why

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Comment by eliykat** [comment]
Waiting until #3382 is merged just to avoid any conflicts.

**Review by ike-kottlowski — APPROVED** [review]

**Pull request merged**
